From ee21c3088aad164207d24bb516756adfdc0642e6 Mon Sep 17 00:00:00 2001
From: Tim Hentenaar <tim@hentenaar.com>
Date: Sat, 7 Feb 2026 22:43:38 -0500
Subject: [PATCH libXmu] test/EditResStream: Fix a comparison in a test case

This happens when running the 32-bit tests on a 64-bit system.
(Issue #4)

Part-of: <https://gitlab.freedesktop.org/xorg/lib/libxmu/-/merge_requests/22>
---
 test/EditResStream.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git ./test/EditResStream.c ./test/EditResStream.c
index 9836236..0162f13 100644
--- ./test/EditResStream.c
+++ ./test/EditResStream.c
@@ -100,7 +100,7 @@ test_EditResStream(void)
 
     res = _XEditResGetWidgetInfo(&ps, &out);
     g_assert_cmpint(res, ==, True);
-    g_assert_cmpmem(ids, sizeof(ids), out.ids, out.num_widgets * sizeof(long));
+    g_assert_cmpmem(ids, sizeof(ids), out.ids, out.num_widgets * sizeof(*out.ids));
     XtFree((char *) out.ids);
     out.ids = NULL;
     _XEditResFreeStream(&ps);
-- 
2.52.0

